### ISSUE TITLE
Document GraphQL max requests per user and any rate causing to exceed limits

### ISSUE THREAD

**Issue by wolruf** [cloudflare/cloudflare-docs#5189]
### Which Cloudflare product does this pertain to?

Analytics

### Existing documentation URL(s)

https://developers.cloudflare.com/analytics/graphql-api/limits/

### Section that requires update

Query Limits

### What needs to change?

Document the max number of GraphQL API per user.

### How should it change?

User should be able to understand why a Dashboard display an error when fetching firewall events.

### Additional information

WAF/Overview Dashboard can display "There was an error fetching your firewall events" due to "rate limiter budget depleted, please try again later"

**Comment by wolruf** [comment]
Let's document if such rates are hard or soft limits and if so, which is the process to raise limits (via CSM for example), if possible of course.

**Comment by victor-perov** [comment]
Hey @wolruf, @angelampcosta!

I apologise for being silent for such a long period.
The problem with current information about limits has been in my head since I added **settings** query, and I believe we should refactor the concept of the limits page completely. I want to push the idea that our users should benefit from introspective query language. Thus, this page should guide our fellow users on how they can use introspection/settings to check what limits they have.

I made room in my task list to tackle this issue. I hope I can provide changes soon.

Kind regards,
Viktor
